src/components/Button/style.css

```diff
@@ -145,6 +145,29 @@
   border-color: var(--rvr-border-color-tertiary-disabled);
 }
 
+.text,
+.text:link,
+.text:visited {
+  color: var(--rvr-color-text);
```

Comment:
Instead of re-declaring the properties and the values, you should be able to declare a component variable that you can re-assign in the scope. For example:

```css
.text {
  --rvr-btn-color: var(--rvr-color-text);
  color: var(--rvr-btn-color); /* sets the default */
}

.text:active {
  /* Changes the variable for the scope, but no property declaration necessary */
  --rvr-btn-color: var(--rvr-color-text-active);
}
```

Reply:
Also, this is not a blocker, but more of a bikeshed thought about how to build out these variants

Reply:
I like this idea, but without spending some time trying to dedupe styles, it doesn't seem like an obvious improvement in this component (since the variables-to-style rules ratio is 1:1).

I'll think about approaches to drying this up, and If I'm missing something, please let me know. For now, I'd like to merge as-is.